app/src/models/FormulaInput.ts

```diff
@@ -160,6 +152,8 @@ export class FormulaInput
           type: DataTypes.UUID,
           allowNull: false,
           field: "formulainput_id",
+          primaryKey: true,
+          defaultValue: DataTypes.UUIDV4,
```

Comment:
We should do this for more tables, much easier to create new instances that way :+1: